src/helpers/ui/paste_interactive.ts

```diff
@@ -8,7 +8,7 @@ export const PasteInteractiveContent = {
     "This operation is not possible due to a merge. Please remove the merges first than try again."
   ),
   wrongFigurePasteOption: _t("Cannot do a special paste of a figure."),
-  frozenPaneOverlap: _t("Cannot paste merged cells over a frozen pane."),
+  frozenPaneOverlap: _t("This operation is not allowed due to an overlapping frozen pane."),
```

Comment:
I made it generalised because while pasting non-merge cells by overlapping a frozen pane also displayed the same msg and I thought it would be misleading for user to display msg with reference to 'merged cells' when it is not the case.

Anyways, I've reverted the change for now :+1:

Reply:
Actually you are 100% correct, I just stopped my search when I saw the code in the merge plugin. I completely misread the use case. the change you made completely makes sense. Sorry :/

Reply:
Okay I'll change it again and push once you're done with reviewing the rest part as well :+1:

Reply:
Changed it back to generalised form like the rest :)